mappings/net/minecraft/village/raid/Raid.mapping

```diff
@@ -25,6 +25,7 @@ CLASS net/minecraft/class_3765 net/minecraft/village/raid/Raid
 	FIELD field_19024 finishCooldown I
 	FIELD field_19172 preCalculatedRavagerSpawnLocation Ljava/util/Optional;
 	FIELD field_30670 MAX_DESPAWN_COUNTER I
+	FIELD field_30675 MAX_RAIDER_DISTANCE_SQUARED I
```

Comment:
```suggestion
	FIELD field_30675 MAX_SQUARED_RAIDER_DISTANCE I
```

Reply:
I think `SQUARED_MAX_RAIDER_DISTANCE` is a more logical ordering.

Reply:
We have `distanceSquared` in `Vec2f`, but `squaredDistanceTo` in most other locations.
I guess haykam's suggestion is the most reasonable here.